src/main/java/com/example/NiRi/UserController.java

```diff
@@ -12,6 +12,9 @@ public class UserController {
     @Autowired
     private UserService userService;
 
+    @Autowired
```

Comment:
can you use constructor autowiring please? https://stackoverflow.com/a/40620318

Reply:
created it.